src/test/groovy/edu/ie3/datamodel/io/factory/result/ConnectorResultFactoryTest.groovy

```diff
@@ -7,13 +7,18 @@ package edu.ie3.datamodel.io.factory.result
 
 import edu.ie3.datamodel.io.factory.SimpleEntityData
 import edu.ie3.datamodel.models.StandardUnits
-import edu.ie3.datamodel.models.result.connector.*
+import edu.ie3.datamodel.models.result.connector.ConnectorResult
+import edu.ie3.datamodel.models.result.connector.LineResult
+import edu.ie3.datamodel.models.result.connector.Transformer2WResult
+import edu.ie3.datamodel.models.result.connector.Transformer3WResult
 import edu.ie3.test.helper.FactoryTestHelper
-import edu.ie3.util.TimeTools
+import edu.ie3.util.TimeUtil
 import spock.lang.Specification
 
 class ConnectorResultFactoryTest extends Specification implements FactoryTestHelper {
 
+	private final TimeUtil timeUtil = TimeUtil.withDefaults
```

Comment:
I would like you to move this object to `FactoryTestHelper` (as a static member). Then you can avoid instantiating it in every factory test.